cookbooks/fb_apache/metadata.rb

```diff
@@ -4,6 +4,7 @@
 license 'BSD'
 description 'Installs/Configures Apache'
 source_url 'https://github.com/facebook/chef-cookbooks/'
+issues_url 'https://github.com/facebook/chef-cookbooks/issues'
```

Comment:
please keep this consistent with the rest of our cookbooks. Since issues URL is consistent with the source_url we have only defined one.